api/v1alpha1/addresspool_types.go

```diff
@@ -23,7 +23,15 @@ import (
 type BgpAdvertisement struct {
 	// The aggregation-length advertisement option lets you “roll up” the /32s into a larger prefix.
 	// +kubebuilder:validation:Minimum=1
-	AggregationLength int `json:"aggregationLength,omitempty" yaml:"aggregation-length,omitempty"`
+	// +kubebuilder:default:=32
```

Comment:
nit about the commit / PR message. The aggregation lenght is added to the BgpAdvertisement API